frontend/common/src/components/UserProfile/UserProfile.tsx

```diff
@@ -93,7 +93,7 @@ const UserProfile: React.FC<UserProfileProps> = ({ applicant, sections }) => {
         {showSection("about") && (
           <TableOfContents.AnchorLink id="about-section">
```

Comment:
Should the id change as well? Any references to `about-section` would also need to be changed.
```suggestion
          <TableOfContents.AnchorLink id="about-me-section">
```